packages/components/modals/ModalBase.tsx

```diff
@@ -159,82 +161,88 @@ const ModalBase: React.FC<ModalBaseProps> = ({
+                        justifyContent: "center",
+                        marginRight: 12,
+                      }}
+                      onPress={onBackPress}
+                    >
+                      <SVG source={chevronLeft} height={12} width={12} />
+                    </TouchableOpacity>
                   )}
 
-                  {labelComponent}
-
-                  {!!description && (
-                    <>
-                      <SpacerColumn size={1} />
-                      <BrandText
-                        style={[
-                          fontSemibold14,
-                          {
-                            color: neutral77,
-                            width: "100%",
-                            lineHeight: 20,
-                            flexWrap: "wrap",
-                          },
-                        ]}
-                      >
-                        {description}
+                  <View style={{ flex: 1, width: "100%" }}>
+                    {label && (
+                      <BrandText style={{ color: "white", lineHeight: 24 }}>
+                        {label}
                       </BrandText>
-                    </>
-                  )}
+                    )}
+
+                    {labelComponent}
+
+                    {description && (
+                      <>
+                        <SpacerColumn size={1} />
+                        <BrandText
+                          style={[
+                            fontSemibold14,
+                            {
+                              color: neutral77,
+                              width: "100%",
+                              lineHeight: 20,
+                              flexWrap: "wrap",
+                            },
+                          ]}
+                        >
+                          {description}
+                        </BrandText>
+                      </>
+                    )}
+                  </View>
                 </View>
-              </View>
-            )}
+              )}
 
-            {!!Header && <Header />}
+              {Header && <Header />}
 
-            <TouchableOpacity
-              containerStyle={[
-                { marginLeft: modalMarginPadding },
-                closeButtonStyle,
-              ]}
-              style={{ justifyContent: "center" }}
-              onPress={onClose}
-            >
-              <SVG width={32} height={32} source={closeSVG} />
-            </TouchableOpacity>
-          </View>
-          {!!children && (
+              <TouchableOpacity
+                containerStyle={[
+                  { marginLeft: modalMarginPadding },
+                  closeButtonStyle,
+                ]}
+                style={{ justifyContent: "center" }}
+                onPress={onClose}
+              >
+                <SVG width={32} height={32} source={closeSVG} />
+              </TouchableOpacity>
+            </View>
+          )}
+          {children && (
```

Comment:
The `!!` ensures the resulting type is a boolean so you have to keep it 👍

Reply:
thks